vagrant-provision.sh

```diff
@@ -0,0 +1,76 @@
+apt-get install python-pip python-dev build-essential default-jre libxslt-dev libxml2-dev libffi-dev libssl-dev git curl -y
+
+# get nodejs from external sources (Debian is outdated)
+curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
+apt-get install nodejs -y
+
+# Install PostgreSQL
+apt-get install postgresql-9.6 postgresql-server-dev-9.6 libpq-dev -y
+su -l postgres -c "psql -c \"CREATE USER indico;\""
+su -l postgres -c "psql -c \"CREATE DATABASE indico WITH OWNER indico;\""
+su -l postgres -c "psql -d indico -c \"CREATE EXTENSION pg_trgm;\""
+su -l postgres -c "psql -d indico -c \"CREATE EXTENSION unaccent;\""
+mv /etc/postgresql/9.6/main/pg_hba.conf /etc/postgresql/9.6/main/pg_hba.conf.default
+sed -i "s:#listen_addresses = 'localhost':listen_addresses = '*':g" /etc/postgresql/9.6/main/postgresql.conf
+echo "
+# TYPE   DATABASE   USER   ADDRESS        METHOD
+local    all        all                   trust
+host     all        all    127.0.0.1/32   trust
+host     all        all    ::1/128        trust
+host     all        all    10.0.0.1/8     trust
+host     all        all    192.186.0.1/16 trust
+" > /etc/postgresql/9.6/main/pg_hba.conf
+service postgresql restart
+
+# Install Redis
+apt-get install redis-server -y
+
+# install Python setup tools and virtualenv
+pip install -U setuptools
+pip install -U virtualenv
+
+# create virtualenv
+sudo -i -u vagrant virtualenv indicoenv
+
+# set some defaults for SSH login
+echo "
+# Python virtualenv for Indico
+source ~/indicoenv/bin/activate
+cd ~/indico/
+" >> /home/vagrant/.profile
+
+# install Python dependencies and Indico
+sudo -i -u vagrant pip install -r requirements.txt
+sudo -i -u vagrant pip install -r requirements.dev.txt
+sudo -i -u vagrant pip install -e .
+
+# create Indico config files
+sudo -i -u vagrant cp indico/logging.conf.sample indico/logging.conf
+sudo -i -u vagrant cp indico/indico.conf.sample indico/indico.conf
+
+# set some required configurations for Indico config file
+cd /home/vagrant/indico
+sed -i 's#/opt/indico/#/home/vagrant/data/#g' indico/indico.conf
+sed -i 's#postgresql:///indico#postgresql://indico@127.0.0.1/indico#g' indico/indico.conf
+sed -i 's#https://indico.yourdomain.tld#http://localhost:8080#g' indico/indico.conf
+sed -i 's:#CustomizationDir:CustomizationDir:g' indico/indico.conf
+sed -i 's:#Debug = False:Debug = True:g' indico/indico.conf
+sed -i "s#SecretKey = ''#SecretKey = '$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)'#g" indico/indico.conf
+sed -i "s,#CeleryBroker = 'redis://unused:password@localhost:6379/0',CeleryBroker = 'redis://localhost:6379/0',g" indico/indico.conf
+
+# create all folders
+mkdir /home/vagrant/data
+mkdir /home/vagrant/data/log
+mkdir /home/vagrant/data/assets
+mkdir /home/vagrant/data/tmp
+mkdir /home/vagrant/data/cache
+mkdir /home/vagrant/data/custom
+chown -R vagrant:vagrant /home/vagrant/data
+
+# do some initial setup for indico
+sudo -i -u vagrant fab setup_deps
```

Comment:
Not needed anymore, since synchronization is done with rsync now (i.e., normale linux file system in the VM). If a network share is set up between host and guest, symlinks don't work and, in consequence, nodeenv doesn't work and everything crashes ;-) since `vagrant rsync-auto` enables a continuous synchronization, I thinkg using rsync is not a big drawback